### DISCUSSION TITLE
Next Teal meetup this week?

### DISCUSSION THREAD

**Discussion by hishamhm** [teal-language/tl#357] [Community]
How does this Thursday, Jan 28 sound like for the next Teal meetup?

Since most attendance last was ranging from (I believe) US East Coast to Europe, we could try to schedule a few hours earlier than last time.

Last time we did it at 6pm UTC ( [click to check it in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20210128T1800) ), so I was thinking if we could make it one or two hours earlier — let us know if that works better, worse or the same for you!

**Comment by ildar** [comment]
Did it happen?
Where's the recording?

**Reply by hishamhm** [reply]
It did not happen, but we're shooting for this Wednesday, Feb 3. We're considering 4pm UTC — feedback on whether this is a better or worse time is welcome!
